World/World/MapViewController.swift

```diff
@@ -0,0 +1,122 @@
+//
+//  MapViewController.swift
+//  World
+//
+//  Created by Timmy Li on 2/11/20.
+//  Copyright © 2020 Timmy Li. All rights reserved.
+//
+
+import UIKit
+import MapKit
+
+//protocol MKMapViewDelegate: class {
+//  /// Function is not implemented in protocol...only defined
+//  func receivedTap(sender: MKMapView)
+//}
+
+class MapViewController: UIViewController {
+    @IBOutlet weak var mapView: MKMapView!
+    @IBOutlet var placeName: UILabel!
+    @IBOutlet var placeDescription: UILabel!
+    @IBOutlet var favoriteStar: UIButton!
+    @IBAction func favoriteStarPress(_ sender: Any) {
```

Comment:
In terms of organizing your project, you can treat this a regular function, therefore moving it below the lifecycle functions.